js/entities/mr-enemy.js

```diff
@@ -10,17 +10,47 @@ class MREnemy extends MREntity {
         this.critSound = document.createElement("mr-entity");
 
         this.poof = document.createElement("mr-model");
+
+        this.el.onLoad = () => {
+
+            // Bind our utils function to the current context here
+            // so we can use the information more easily.
+
+            const boundUpdateClipsFor = updateClipsFor.bind(this.el);
+
+            // Cleanup animation clips for enemy
+
+            switch (this.subtype) {
+            case 'static':
+                console.log('do something');
+                break;
+            case 'homing':
+                console.log('do something');
+                break;
+            case 'aimless':
+            default: 
+                boundUpdateClipsFor('idle', 1, 30);
+                boundUpdateClipsFor('attack', 31, 45);
+                boundUpdateClipsFor('crit', 46, 60);
+                boundUpdateClipsFor('damage', 61, 75);
+            }
```

Comment:
added in this switch statement in case you wanted it to be different based on this.subtype (for what animations / clips need to load)